recipes/freealut/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, collect_libs, export_conandata_patches, copy, get, rmdir, rm
+import os
+
+required_conan_version = ">=1.54.0"
+
+class FreeAlutConan(ConanFile):
+    name = "freealut"
+    description = "freealut is a free implementation of OpenAL's ALUT standard."
+    topics = ("freealut", "openal", "audio", "api")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://directory.fsf.org/wiki/Freealut"
+    license = "LGPL-2.0-or-later"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True]
+    }
+    default_options = {
+        "shared": True
+    }
```

Comment:
```suggestion
    package_type = "shared-library"
    settings = "os", "arch", "compiler", "build_type"
```

Reply:
Since the most recipes build as static by default (including the openal-soft), i did like the grpc recipe. Allow it to build static, but if the freealut is built as shared lib it forces to use shared requirements to avoid binary duplication.